infra/discovery/src/listener/listener.js

```diff
@@ -114,9 +114,14 @@ async function main() {
   // List of contracts the listener watches events from.
   const contracts = {
     IdentityEvents: contractsContext.identityEvents,
-    Marketplace: contractsContext.marketplaces['000'].contract,
     ProxyFactory: contractsContext.ProxyFactory
   }
+  // Listen to all versions of marketplace
+  Object.keys(contractsContext.marketplaces).forEach(key => {
+    console.log('key', key)
```

Comment:
Can we take out the log?

Reply:
woops, yeah, I'll fix that before merge.  Thanks